vault-plugins/flant_iam/model/role.go

```diff
@@ -62,6 +71,8 @@ type Role struct {
 	RequireOneOfFeatureFlags []FeatureFlagName `json:"require_one_of_feature_flags"`
 	IncludedRoles            []IncludedRole    `json:"included_roles"`
 
+	ArchivedAt    UnixTime `json:"archived_at"`
```

Comment:
More declarative name would be "archiving_timestamp". Like in kubernetes naming conventions.